.github/workflows/releases.yml

```diff
@@ -0,0 +1,34 @@
+name: Releases
+on:
+  workflow_dispatch:
+
+jobs:
+  publish_release:
+    runs-on: [self-hosted, Linux]
+    container:
+      image: lampepfl/dotty:2021-03-22
+      options: --cpu-shares 4096
+      volumes:
+        - ${{ github.workspace }}/../../cache/sbt:/root/.sbt
+        - ${{ github.workspace }}/../../cache/ivy:/root/.ivy2/cache
+        - ${{ github.workspace }}/../../cache/general:/root/.cache
```

Comment:
It is not clear to me why we need this? It seems that all we do is some cURLing, right?

Reply:
Right, it is unnecessary as well as the next steps you mentioned